app/models/planet.py

```diff
@@ -0,0 +1,15 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String, nullable = False)
+    description = db.Column(db.String, nullable = False)
+    size = db.Column(db.String, nullable = False)
+
+    def to_json(self):
```

Comment:
An easy change, but like it was mentioned in Roundtable, this method returns a dictionary so a better name would be `to_dict`